src/sphere.py

```diff
@@ -0,0 +1,63 @@
+import math
+
+
+class Sphere:
+    """
+    Class for managing Spheres within a 3D space.
+    """
+
+    def __init__(self, centre_point, radius):
```

Comment:
Why no type hints?

Reply:
I added the ones I could, buy adding more requires `typing`. That said, if I am introducing `unittest`, I could do `typing` with it like I did in the original testing and type hints topic early in the course.

Reply:
`float` was added, but to use `Point3D`, I need typing

```python
from typing import Type
...
    def __init__(self, centre_point: Type[Point3D], radius: float):
```

Reply:
[nope](https://github.com/duck-dynasty/duckbot/blob/main/duckbot/slash/slash_command_handler.py#L17)

Reply:
my life has been a lie

Reply:
[Type[T]](https://docs.python.org/3/library/typing.html#typing.Type) is for generic types. Classes, you can just use use the class name.

Reply:
Gonna' go back and add type hints to previous classes now.